core/save-profiledata.c

```diff
@@ -21,6 +22,15 @@ static void put_double(struct membuffer *b, double val)
 	put_format(b, "\"%f\" ", val);
 }
 
+static void put_secs(struct membuffer *b, int secs)
```

Comment:
that seems oddly named - I would assume that this writes the # of seconds. Isn't this much more `put_timestamp()`?

Reply:
It does, but in the format the subtitle file likes. Will rename if you prefer.

Reply:
I would prefer if you'd rename it. We should try to have reasonable function names, especially when introducing something new (so we don't fight with any legacy understanding)